ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -18,22 +16,40 @@
  */
 public class AddFormTest extends TestBase {
     private HomePage homePage;
+    private static String name = "newFormTest";
+    private static String description = "description of new form";
+    private static  String version = "1.2";
+
+    private static final By NEW_HTML_FORM = By.cssSelector("a[href='htmlForm.form']");
+
+
     private HeaderPage headerPage;
     private ManageFormsPage manageForm;
     private ClinicianFacingPatientDashboardPage patientDashboardPage;
+    private AdministrationPage administrationPage;
+    private ManageHtmlFormsPage manageHtmlFormsPage;
+    private HtmlFormsPage htmlFormsPage;
 
     @Before
     public void setUp() throws Exception {
         homePage = new HomePage(page);
-        assertPage(homePage);
         headerPage = new HeaderPage(driver);
+        administrationPage = new AdministrationPage(page);
         manageForm = new ManageFormsPage(driver);
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
+
     }
 
-    @Ignore //ignore due to moving forms functionality
     @Test
     public void addFormTest() throws Exception {
+     manageHtmlFormsPage =  homePage.goToAdministration().clickOnManageHtmlForms();
+
+     if (manageHtmlFormsPage.getElementsIfExisting(By.xpath("//*[contains(text(), '"+ name+ "')]")).isEmpty()){
```

Comment:
try rightly formatting your code to well format `+ name+ `

Reply:
Resolved thanks